app/markdown/lib/parser/original/markdown.js

```diff
@@ -103,7 +103,8 @@ const parseNotEscaped = (message, {
 		const target = url.indexOf(rootUrl) === 0 ? '' : '_blank';
 		title = title.replace(/&amp;/g, '&');
 
-		const escapedUrl = encodeURI(url);
+		const escapedUrl = /\%/i.test(url) ? url : encodeURI(url);
```

Comment:
I believe we introduced this encode to prevent some html injections, passing `%` disable this protection, please check if there is any better way to do that

Reply:
I found a solution, instead of checking for the percentage symbol in the URL . I just decoded the URL before encoding it. This will make sure all decoded URLs are encoded, as well as already encoded URLs are not encoded again.

Reply:
@ggazzo can you approve ?